src/main/kotlin/org/rust/cargo/toolchain/flavors/RsToolchainFlavor.kt

```diff
@@ -17,8 +18,22 @@ abstract class RsToolchainFlavor {
 
     fun suggestHomePaths(): Sequence<Path> = getHomePathCandidates().filter { isValidToolchainPath(it) }
 
+    fun suggestBazelPaths(projectPath: Path?): Sequence<Path> {
```

Comment:
In a Bazel project, mandate the usage of a Bazel-installed Rust toolchain (even `rustup` is not allowed!)